### PR TITLE
Allow for the slash character in command line input values

### PR DESCRIPTION
**Pull request by nmsriram**
## PR Summary

This PR changes the logic of the parsing of inputs provided on the command line to allow users to specify a `/` in values for keys on the command line.  This is useful when specifying a directory for output or a data file which is in a different directory structure.

## PR Checklist

- [ ] Code passes cpplint
- [ ] New features are documented.
- [ ] Adds a test for any bugs fixed. Adds tests for new features.
- [ ] Code is formatted
- [ ] Changes are summarized in CHANGELOG.md
- [x] (@lanl.gov employees) Update copyright on changed files

### CODE REVIEW

**Comment by JoshuaSBrown** [comment]
Can you give me the use case where this was a problem, just for documentation? Otherwise I don't really see a problem with this pr.

File: cmake/CheckCopyright.cmake [hunk 1]
```diff
@@ -89,6 +89,7 @@ foreach(INPUT ${COPYRIGHTABLE})
         COMMAND
             ${CMAKE_COMMAND} -P ${CMAKE_CURRENT_LIST_DIR}/CheckCopyrightScript.cmake
                 update ${CMAKE_CURRENT_SOURCE_DIR} ${GIT_EXECUTABLE} ${INPUT})
+>>>>>>> develop
```
Comment (JoshuaSBrown) [inline cmake/CheckCopyright.cmake]: might want to remove this ;)

**Comment by nmsriram** [comment]
> Can you give me the use case where this was a problem, just for documentation? Otherwise I don't really see a problem with this pr.

For one of my projects - converting input files from one code to another I need to specify the input file as a parameter.  This change allows me to run through several input files by defining them at the command line.

**Review by JoshuaSBrown — APPROVED** [review]
Ok, I see what you did, this is an improvement thanks.

File: src/parameter_input.cpp [hunk 2]
```diff
@@ -401,8 +401,8 @@ void ParameterInput::ModifyFromCmdline(int argc, char *argv[]) {
 
   for (int i = 1; i < argc; i++) {
     input_text = argv[i];
-    std::size_t slash_posn = input_text.rfind("/");         // find last "/" character
-    std::size_t equal_posn = input_text.find_first_of("="); // find "=" character
+    std::size_t equal_posn = input_text.find_first_of("=");     // first "=" character
+    std::size_t slash_posn = input_text.rfind("/", equal_posn); // last "/" before "="
```
Comment (Yurlungur) [inline src/parameter_input.cpp:405]: Clever.

**Review by Yurlungur — APPROVED** [review]
LGTM.

**Pull request merged**